apps/google-analytics-4/lambda/config/serverless-env.test.yml

```diff
@@ -1 +1,2 @@
-SIGNING_SECRET: ${env:SIGNING_SECRET}
+SIGNING_SECRET: ${ssm:/aws/reference/secretsmanager/ci/apps/google-analytics-4-action/test/signing-secret}
+domainName: google-analytics-4-test.ctfapps.net
```

Comment:
it's pretty common convention for env vars is to be uppercased. is there a reason we're doing these in camelCase?